docs/react-native/connecting.mdx

```diff
@@ -11,21 +11,22 @@ How to connect to Fishjam Cloud
 
 ## Getting URL and token
 
-In order to connect, you need to retrieve URL (multimedia server address) and token (token that will authenticate you in your Room)
+In order to connect, you need to retrieve URL (multimedia server address) and token (token that will authenticate you in
+your Room)
 
 <Tabs groupId="app-type">
 
   <TabItem value="sandbox" label="Sandbox App">
 
-Once you get your account on [Fishjam Cloud](https://fishjam.io), you will have access to Sandbox App.
-This app comes with pre-configured test service called Room Manager. This is basically service that will
-create Room, add your app as Room's Participant and return token required to use that Room.
+Once you get your account on [Fishjam Cloud](https://fishjam.io), you will have access to Sandbox App. This app comes
+with pre-configured test service called Room Manager. This is basically service that will create Room, add your app as
+Room's Peer and return token required to use that Room.
```

Comment:
**note (non-blocking):** This sentence is unclear, but let's fix it in another PR.